package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "bookstore-add-reducers-and-actions",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
- [ ] Per the project requirements, `@reduxjs/toolkit` & `react-redux` should be present in package.json. Kindly install these two packages from `npm`.

Reply:
Thank you I will do that.